### PR TITLE
Remove unused copy of xcode files during iOS build

### PR DESCRIPTION
**Pull request by tvolkert**

### CODE REVIEW

File: sky/shell/BUILD.gn [hunk 1]
```diff
@@ -389,8 +388,7 @@ if (is_android) {
     ]
   }
 
-  sky_ios_sdk("shell") {
-    sdk_name = "Flutter"
+  group("shell") {
   }
```
Comment (abarth) [inline sky/shell/BUILD.gn]: We probably need to make this depend on something in order to trigger Flutter.framework to build on iOS

Comment (tvolkert) [inline sky/shell/BUILD.gn]: Hm, it's still building the Flutter framework - I can try to see where it's picking up the dependency.  Previously, it only depended upon the two copies:

```
group(target_name) {
  deps = [
    ":copy_sdk_xcode_harness",
    ":copy_user_editable_files",
  ]
}
```

Comment (abarth) [inline sky/shell/BUILD.gn]: Looks like there's a dependency directly from the root BUILD.gn file.

**Comment by abarth** [comment]
LGTM

**Pull request merged**